### ISSUE TITLE
Support for pandoc "native divs and spans" in export to markdown (e.g. for reveal.js speaker notes)

### ISSUE THREAD

**Issue by bgmilne** [zim-desktop-wiki/zim-desktop-wiki#1031]
I am playing with using zim->pandoc->reveal.js (among others) as a mechanism to provide documentation for my team that is:
* accessible and quickly searchable on a desktop/laptop (using zim)
* available on our wiki pages (using pandoc to convert to the wiki format)
* available as build artifacts (exporting using zim)
* usable for fancy presentations (using reveal.js) (besides the existing Slides template which isn't as nice for actual presentation on a projector).

I have followed the approach [here](https://github.com/jgm/pandoc/wiki/Using-pandoc-to-produce-reveal.js-slides), but when trying to add [speaker notes](https://github.com/hakimel/reveal.js#speaker-notes) I haven't been successful in trying to using the method in the [pandoc](https://pandoc.org/MANUAL.html#speaker-notes) documentation.

If I add something like this in my zim wiki file:
```
::: notes

This is a speaker note

:::
```

Then when exporting from zim to pandoc, this is omitted entirely.

This format [seems to be referred to as *divs and spans*](https://pandoc.org/MANUAL.html#divs-and-spans), and there are other useful uses of this feature as shown there, and in the rest of the manual (just search for `:::`), such as [language variables](https://pandoc.org/MANUAL.html#language-variables), which may be of interest with respect to https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/961 .

If there is a way to do this, it should be added to the documentation.

**Comment by jaap-karssenberg** [comment]
Just ran a quick test and what I see is that such speaker notes pass
through zim to the pandoc output unchanged - which is what you want.

Also I cannot think of any reason why zim would omit them in the export
since for zim the ":::" has no special meaning.

Can you add an example of a page where the export fails, or more detailed
instructions how to reproduce?

-- Jaap

**Comment by bgmilne** [comment]
Weird.

I created a new notebook to make a reproducible case, and it worked.

Then I went back to the notebook where I ran into this problem, and added the:

```
::: notes
```

and

```
:::
```

lines, and it also worked.

Nevertheless, I have attached a [tarball](https://github.com/zim-desktop-wiki/zim-desktop-wiki/files/4328340/ZimRevealJS.tar.gz) of the test notebook I created, which contains the instructions for creating a RevealJS slide show.
